homeassistant/components/elmax/config_flow.py

```diff
@@ -170,25 +177,29 @@ async def async_step_panels(
 
     async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
         """Perform reauth upon an API authentication error."""
-        self._reauth_username = entry_data.get(CONF_ELMAX_USERNAME)
         self._reauth_panelid = entry_data.get(CONF_ELMAX_PANEL_ID)
         return await self.async_step_reauth_confirm()
 
-    async def async_step_reauth_confirm(self, user_input=None):
+    async def async_step_reauth_confirm(self, user_input=None) -> FlowResult:
         """Handle reauthorization flow."""
         errors = {}
         if user_input is not None:
-            panel_pin = user_input.get(CONF_ELMAX_PANEL_PIN)
+            username = user_input.get(CONF_ELMAX_USERNAME)
             password = user_input.get(CONF_ELMAX_PASSWORD)
+            panel_pin = user_input.get(CONF_ELMAX_PANEL_PIN)
```

Comment:
Is required according to schema so no need to use .get

Reply:
Addressed in last commit